src/main/webapp/app/assessment/assessment-filters/assessment-filters.component.html

```diff
@@ -1,10 +1,12 @@
-<div class="d-flex align-items-center my-2">
-    <label class="radio-inline mb-0 d-flex align-items-center">
-        <input type="radio" [ngModel]="filterProp" (click)="updateFilter(FilterProp.ALL)" [value]="FilterProp.ALL" />
-        <span class="ms-1">{{ 'artemisApp.assessment.dashboard.filters.showAll' | artemisTranslate }}</span>
-    </label>
-    <label class="radio-inline ms-2 mb-0 d-flex align-items-center">
-        <input type="radio" [ngModel]="filterProp" (click)="updateFilter(FilterProp.LOCKED)" [value]="FilterProp.LOCKED" />
-        <span class="ms-1">{{ 'artemisApp.assessment.dashboard.filters.showLocked' | artemisTranslate }}</span>
-    </label>
-</div>
+<ng-container *ngIf="submissions?.length && filteredSubmissions > 0">
```

Comment:
Is there any particular reason why you put the `*ngIf` inside of an `ng-container` instead of directly putting it into the `div`?
It should behave the same but avoids an unnecessary indentation.